### PR TITLE
Unify Integration Update zip file

### PR DESCRIPTION
**Pull request by mary-element**
### Changes:
Update Unify lambda zip files.  

script  "unify_scripts/deploy_to_repo.sh"  build zip files from the source code and upload to test repo "s3://imc-unify-demo/"

### How to test 

1. run "unify_scripts/deploy_to_repo.sh" to build and deploy zip file to test repo "imc-unify-demo"
2. Create stacks from test repo
3. Make sure the lamdba functions works and zip code have the right version

### CODE REVIEW

**Comment by sshvans** [comment]
Please address the linting issues, before I can merge the PR.

File: templates/Physical.template.yaml [hunk 1]
```diff
@@ -240,7 +240,7 @@ Resources:
         Fn::GetAtt:
           - BucketNotificationRole
           - Arn
-      Runtime: nodejs10.x
+      Runtime: nodejs14.x
```
Comment (mary-element) [inline templates/Physical.template.yaml:243]: @jieatelement we need to test if this code still works with nodejs14

**Comment by jieatelement** [comment]
@sshvans - There are some remaining lint errors that are blocking the PR from merging. Mary has addressed most of these, but the remaining lint errors are for files that our team did not touch as part of the IMC integration with Unify. We want to do a test to make sure the templates all work, but please advises on the remaining lint errors.

https://github.com/aws-quickstart/quickstart-aws-industrial-machine-connectivity/pull/55/checks?check_run_id=2879254226

File: templates/IMC-unify-main.template.yaml [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+        PublicSubnet1CIDR: !Ref PublicSubnet1CIDR
+        VPCCIDR: !Ref VPCCIDR
+        VPCTenancy: !Ref VPCTenancy
+        NumberOfAZs: 3
+
+  Workload:
+    Type: 'AWS::CloudFormation::Stack'
+    Properties:
+      TemplateURL: !Sub
+        - https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}templates/IMC-workload.template.yaml
```
Comment (jieatelement) [inline templates/IMC-unify-main.template.yaml:232]: @mary-element IMC-unify-workload.template.yaml

Comment (mary-element) [reply templates/IMC-unify-main.template.yaml]: resovled

File: templates/Physical.template.yaml (hunk 1 above)
Comment (jieatelement) [reply templates/Physical.template.yaml]: Already reverted back to nodejs10.x. Adding a note, nodejs14.x does not support ZipFile when testing.

**Comment by sshvans** [comment]
@mary-element @jieatelement Thanks for the PR! Can you please pull the latest from the develop branch and resolve the merge conflicts? After the merge conflicts are resolved, you shouldn't see any cfn-lint issues in the non-unify templates, as those are all resolved in the develop branch.

**Comment by mary-element** [comment]
Hi @sshvans, I resolved the conflicts. 
There are still some lint errors from non-unify templates:  https://github.com/aws-quickstart/quickstart-aws-industrial-machine-connectivity/pull/55/checks?check_run_id=3049366007 though.

**Review by sshvans — COMMENTED** [review]
Merging to test lint issues

**Pull request merged**